packages/api/src/store/asset-table.ts

```diff
@@ -1,6 +1,7 @@
+import _ from "lodash";
```

Comment:
I am not getting in what context `lodash` is used

Reply:
Only as a little helper when manipulating objects https://github.com/livepeer/studio/pull/1178/files/171bf9964a3088f5c0ffe8fc7a70dfa2763481ba#diff-d7e49e7719145cd61bdb5994019c95d2abb9c9fe8de1367dd4fb1ef97eb0e33dR91